src/vic3/effect_validation.rs

```diff
@@ -555,18 +556,77 @@ pub fn validate_create_military_formation(
 
 pub fn validate_create_pop(
     _key: &Token,
-    _block: &Block,
+    block: &Block,
     _data: &Everything,
     _sc: &mut ScopeContext,
     mut vd: Validator,
     _tooltipped: Tooltipped,
 ) {
     // This effect is undocumented
 
-    vd.field_item("culture", Item::Culture);
-    vd.field_item("religion", Item::Religion);
     vd.field_item("pop_type", Item::PopType);
     vd.field_integer("size");
+
+    // TODO: Consider doing sanity checks on the numbers in the splits
+    //  Since they're percentages of the size, not ratios, if they don't add up to 1 you'll get unexpected results
+    //  But that would involve float maths and comparisons for something that isn't technically wrong.
+
+    let mut available_cultures = HashSet::new();
+
+    if block.has_key("culture") {
+        vd.ban_field("cultures", || "pops with several cultures");
+        vd.field_item("culture", Item::Culture);
+        if let Some(token) = vd.field_value("culture") {
+            available_cultures.insert(token.as_str());
+        }
+    } else {
+        vd.field_validated_block("cultures", |block, data| {
+            let mut vd = Validator::new(block, data);
+            vd.validate_item_key_values(Item::Culture, |key, mut vd| {
+                available_cultures.insert(key.as_str());
+                vd.numeric()
```

Comment:
`numeric_range(0.0..=1.0)` would be nice here. It doesn't exist yet in ValueValidator, but Validator has the corresponding field function